client/jetpack-connect/plans.jsx

```diff
@@ -145,59 +145,54 @@ class Plans extends Component {
 	}
 
 	autoselectPlan() {
-		if ( ! this.props.showFirst ) {
```

Comment:
Unfortunate diff below, but it should just be the formatting changes resulting from the removal if the above `if`.

Reply:
Thanks for the note, that an inflated diff 😵